src/logistration/LoginFailure.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+
+import { Alert, Hyperlink } from '@edx/paragon';
+
+const processLink = (link) => {
+  let matches;
+  link.replace(/(.*)<a href=["']([^"']*).*>([^<]+)<\/a>(.*)/g, function () { // eslint-disable-line func-names
```

Comment:
Can we move that expression to constant's or at least assign to a variable ?

Reply:
This has single usage, so i kept it like this. Name of the function do explain what the regex is doing.